hypha/apply/api/v1/review/permissions.py

```diff
@@ -79,3 +79,15 @@ def has_object_permission(self, request, view, obj):
             return True
 
         return False
+
+
+class HasReviewDraftPermission(permissions.BasePermission):
+    """
+    Custom permission that user should have for editing review.
+    """
+    def has_object_permission(self, request, view, obj):
+        try:
+            submission = view.get_submission_object()
+        except KeyError:
+            return True
```

Comment:
Is not this should be false in case of error? No permission in case of errors.

Reply:
This would never happen in an ideal case. This is just to avoid error when accessing the doc.

If you access a url without submission id, it will give you 404.